courses/views_test.py

```diff
@@ -115,62 +222,132 @@ def test_delete_program(user_drf_client, programs):
     assert resp.status_code == status.HTTP_405_METHOD_NOT_ALLOWED
 
 
-def test_get_courses(user_drf_client, courses, mock_context):
+@pytest.mark.parametrize("num_courses", [100])
+@pytest.mark.parametrize("num_programs", [15])
```

Comment:
@rhysyngsun since it really is a function and not a fixture, I opted instead to just parameterize two variables for the test